plasma_framework/migrations/1030_output_results.js

```diff
@@ -14,12 +14,19 @@ module.exports = async (
     // eslint-disable-next-line no-unused-vars
     [deployerAddress, maintainerAddress, authorityAddress],
 ) => {
+    let authority;
+    const vault = process.env.VAULT || false;
+    if (vault) {
+        authority = fs.readFileSync('vault_authority');
+    } else {
+        authority = authorityAddress;
+    }
     const plasmaFramework = await PlasmaFramework.deployed();
     const ethVault = await plasmaFramework.vaults(config.registerKeys.vaultId.eth);
     const erc20Vault = await plasmaFramework.vaults(config.registerKeys.vaultId.erc20);
     const paymentExitGame = await plasmaFramework.exitGames(config.registerKeys.txTypes.payment);
```

Comment:
looks like `paymentExitGame`, `erc20Vault` and `ethVault` get a value of address(0) here since the deployment is on 1040 and 1041. Maybe logging there would be a good idea?

Reply:
ah yeah, 1030 should be moved down the road.

Reply:
or removed, we're not using these results anymore I think.

Reply:
Just logging the deployed contract addresses, should be good to have